Lombiq.NodeJs.Extensions/scripts/copy-assets.js

```diff
@@ -9,55 +9,68 @@ const util = require('util');
+logLine(`Started executing copy-assets.js at "${projectPath}".`);
 
-async function copyFilesFromConfig(config) {
+function copyFilesFromConfig(config) {
     return Promise.all(config
         .map((assetsGroup) => assetsGroup.sources.map((assetSource) => {
             // Normalize the relative path to the directory to remove trailing slashes and straighten out any anomalies.
-            const directoryToCopy = path.normalize(assetSource);
+            const directoryToCopy = path.normalize(path.resolve(projectPath, assetSource));
             const pattern = assetsGroup.pattern;
-
             logLine(`Copy assets from "${directoryToCopy}" using pattern "${pattern}"...`);
 
-            return access(directoryToCopy)
-                .then(
-                    () => {
-                        const pathPattern = path.join(directoryToCopy, pattern);
-                        const sourceAndTargetPaths = [pathPattern, assetsGroup.target];
+            return access(directoryToCopy).then(
+                () => {
+                    const pathPattern = path.join(directoryToCopy, pattern);
+                    const targetPath = (process.platform === 'win32')
+                        ? assetsGroup.target
+                        : path.normalize(path.resolve(projectPath, assetsGroup.target));
+                    const sourceAndTargetPaths = [pathPattern, targetPath];
 
-                        // We want to copy all files matched by the given pattern into the target folder mirroring the
-                        // source folder structure. This is done by removing the source folder path from the beginning
-                        // which "copyfiles" does using the "up" option.
-                        const depth = directoryToCopy.split(/[\\/]/).length;
+                    // We want to copy all files matched by the given pattern into the target folder mirroring the
+                    // source folder structure. This is done by removing the source folder path from the beginning
+                    // which "copyfiles" does using the "up" option.
+                    const depth = directoryToCopy.split(/[\\/]/).length;
 
-                        // See https://github.com/calvinmetcalf/copyfiles#programic-api for more details.
-                        return copyfiles(sourceAndTargetPaths, { verbose: verbose, up: depth }, () => {});
-                    },
-                    () => handleErrorObject({
-                        code: 'NE0031',
-                        path: 'AssetCopy',
-                        message: `The directory "${directoryToCopy}" cannot be accessed to copy files from.`,
-                    }));
+                    // See https://github.com/calvinmetcalf/copyfiles#programic-api for more details.
+                    return copyfiles(sourceAndTargetPaths, { verbose: verbose, up: depth }, () => {});
+                },
+                () => handleErrorObject({
+                    code: 'NE31',
```

Comment:
Is this intended? `NE0031` -> `NE31`

Reply:
Yes. All other Lombiq.NodeJs.Extensions error codes are in the NExx format were xx is 2 digits, so I'm sure this was a mistake.